var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -313,14 +324,15 @@ def cmake_args(self):
         from_variant = self.define_from_variant
 
         if spec.satisfies("~wrapper+cuda") and not (
-            spec.satisfies("%clang") or spec.satisfies("%cce")
+            spec.satisfies("%clang") or spec.satisfies("%cce") or spec.satisfies("+cmake_lang")
```

Comment:
You may also update the error message below.

Reply:
Updated.